src/shared/infra/http/config/middlewares.ts

```diff
@@ -1,8 +1,10 @@
 import { Express } from 'express';
 import { bodyParser, contentType, cors } from '../middlewares';
+import helmet from 'helmet';
 
 export default (app: Express): void => {
   app.use(bodyParser);
   app.use(cors);
   app.use(contentType);
+  app.use(helmet());
```

Comment:
so, the helmet() is a middlware, the correct one then is you create your own file for it in `src/shared/infra/http/middleware`, and after that import this module in that file, take the example of bodyParser